### ISSUE TITLE
Missing initial instructions in evals/benchmark/README (and others)

### ISSUE THREAD

**Issue by dbkinder** [opea-project/GenAIEval#92]
In GenAIEval/evals/benchmark/README.md (and others), the first step in the instructions is, 

```
pip install -r ../../requirements.txt
```

There's a non-stated assumption of the current directory before this step is executed.  How did the requirements.txt file get on the developer's computer?  There's no instruction to clone the GenAIEval repo locally, for example.

**Comment by mixmaier** [comment]
I agree with the above statement. I am also trying to use benchmarking tools but have problems to figure out how to...

**Comment by yinghu5** [comment]
@mixmaier @dbkinder @feng-intel 
Thank for raising and addressing the problem. you are welcomed to try the instructions: https://opea-project.github.io/latest/GenAIEval/README.html and feedback if any issues.
